compiler/rustc_resolve/src/imports.rs

```diff
@@ -321,6 +321,11 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                         } else if !old_binding.vis.is_at_least(binding.vis, this.tcx) {
```

Comment:
```suggestion
                        } else if binding.is_ambiguity() || !old_binding.vis.is_at_least(binding.vis, this.tcx) {
```
The `matches!(..., NameBindingKind::Import { .. })` conditions are redundant, we are already checking it above.

Reply:
Oh yes, I just realized that I had already run ﻿`binding.is_glob_import`.